src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/help.py

```diff
@@ -91,3 +91,11 @@
             type: group
             short-summary: Command to manage databases activities in database elastic pools
             """
+helps['sql elastic-pools recommended'] = """
+            type: group
+            short-summary: Commands to see information about an Azure SQL Recommended Elastic Pools
```

Comment:
nit: change 'Pools' to 'Pool' (consistent with the 'elastic-pools recommended db' text)